libs/pageserver_api/src/models.rs

```diff
@@ -979,11 +979,32 @@ impl PagestreamFeMessage {
         //
         // TODO: consider using protobuf or serde bincode for less error prone
         // serialization.
-        let msg_tag = body.read_u8()?;
+        let mut msg_tag = body.read_u8()?;
+        //
+        // Old version of protocol use commands with tags started with 0 and containing `latest` flag.
+        // New version of protocol shift command tags by 10 and pass LSN range instead of `latest` flag.
+        // Server should be able to handle both protocol version. As far as we are not passing no=w,
+        // protocol version from client to server, we make a decision based on tag range.
+        // So this code actually provides backward compatibility.
```

Comment:
```suggestion
        // Server should be able to handle both protocol versions. As far as we are not passing now
        // protocol version from client to server, we decide based on tag range.
        // So, this code provides backward compatibility.
```

I was first thinking the `no=w` was some assignment which I couldn't figure out, but perhaps it was just a typo.